numismatic/feeds/base.py

```diff
@@ -245,52 +249,33 @@ class WebsocketClient(abc.ABC):
 
     exchange = None
     websocket_url = None
-    _websocket_map = dict()
-    _semaphore_map = dict()
-
-    @property
-    def websocket(self):
-        try:
-            return self._websocket_map[self.websocket_url]
-        except KeyError:
-            return None
-    
-    @property
-    def semaphore(self):
-        if self.websocket_url is None:
-            return None
-        try:
-            return WebsocketClient._semaphore_map[self.websocket_url]
-        except KeyError:
-            semaphore = asyncio.BoundedSemaphore(1)
-            WebsocketClient._semaphore_map[self.websocket_url] = semaphore
-            return semaphore
-        except Exception as ex:
-            logger.error(ex)
-            logger.error(packet)
-            raise
-
-    async def _connect(self):
+    _websocket_future = None
+    __listener = None
+    subscriptions = []
+            
+    def __attrs_post_init__(self):
+        asyncio.ensure_future(self._listener())
+
+    async def _connect(self, websocket_url=None):
```

Comment:
I've removed the maps because I think a single `WebsocketClient` should be responsible for a single `Websocket`. Luno's use of multiple websockets should be handled in their feed.